ibm/service/iampolicy/data_source_ibm_iam_policy_template.go

```diff
@@ -187,10 +187,62 @@ func DataSourceIBMIAMPolicyTemplate() *schema.Resource {
 									},
 									"roles": {
 										Type:        schema.TypeList,
-										Required:    true,
+										Optional:    true,
 										Elem:        &schema.Schema{Type: schema.TypeString},
 										Description: "Role names of the policy definition",
 									},
+									"role_template_references": {
+										Type:        schema.TypeList,
+										Optional:    true,
+										Description: "Role template references for assignment.",
+										Elem: &schema.Resource{
+											Schema: map[string]*schema.Schema{
+												"version": {
+													Type:        schema.TypeString,
+													Computed:    true,
+													Description: "Role template version",
+												},
+												"id": {
+													Type:        schema.TypeString,
+													Computed:    true,
+													Description: "Role template id",
+												},
+											},
+										},
+									},
+									"subject": {
```

Comment:
If this was missing before, was data source working correctly with authorization templates?